src/components/network/network-table.js

```diff
@@ -421,6 +442,22 @@ const NetworkTable = (props) => {
                     );
                 });
             }
+
+            if (isModifyingRow()) {
+                // When modifying a row, a shallow copy of the edited row is created and put on top of the table.
+                // The edition is then done on the copy, on top of the table, and not below, to prevent
+                // bugs of rows changing position in the middle of modifications.
+                let editRowPosition;
```

Comment:
I wonder if we could have memorized the rowPosition when we click on edition button (cf setLineEdit), rather than compute it.